packages/SwingSet/src/liveslots/virtualObjectManager.js

```diff
@@ -140,7 +140,7 @@ export function makeCache(size, fetch, store) {
  *   of virtual references.
  * @param {() => number} allocateExportID  Function to allocate the next object
  *   export ID for the enclosing vat.
- * @param { (val: Object) => string} getSlotForVal  A function that returns the
+ * @param {(val: object) => string} getSlotForVal  A function that returns the
```

Comment:
I believe it's the same here, `val` would now become an `any` type.

Reply:
> now become an `any` type

Again, already was.